zerver/tests/test_link_embed.py

```diff
@@ -30,10 +30,6 @@
         "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
         "LOCATION": "url-preview",
     },
-    "in-memory": {
-        "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
-        "LOCATION": "url-preview",
-    },
```

Comment:
The commit message discussion about how we've solved this problem in the test system isn't quite right -- we now generate a random `KEY_PREFIX` via `bounce_key_prefix_for_testing` for each individual test.

Reply:
The `in-memory` cache was never about the test system -- it's about the standalone development server.  My point was that while `./tools/run-dev.py` clears memcached when it starts up, `./manage.py runserver` doesn't clear memcached (or rotate `KEY_PREFIX`) when it hot-reloads new code.